meinberlin/apps/contrib/tasks.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from pathlib import Path
+
+import requests
+from celery import shared_task
+from django.conf import settings
+from django.core.cache import cache
+
+logger = logging.getLogger(__name__)
+
+
+def download_footer(footer_url, filepath):
+    try:
+        response = requests.get(footer_url, timeout=2.0)
+        filepath.write_bytes(response.content)
+        # clear cache
+        cache.delete("footer")
```

Comment:
why delete the cache here? we are updating it in `periodic_footer_update` anyway. And if the download fails we don't want it deleted because otherwise the footer will disappear then?